analytics-api/app/controllers/JobController.scala

```diff
@@ -77,16 +77,16 @@ class JobController @Inject() (
 
   def getTelemetry(datasetId: String) = Action.async { request: Request[AnyContent] =>
 
-    val summaryType =  request.getQueryString("type")
-    val from = request.getQueryString("from").getOrElse("")
+    val since = request.getQueryString("since").getOrElse("")
+    val from = if (since.nonEmpty) since else request.getQueryString("from").getOrElse(org.ekstep.analytics.api.util.CommonUtil.getPreviousDay())
```

Comment:
Should have moved this logic to the API service layer

Reply:
Have moved this logic to service layer